### DISCUSSION TITLE
Request to join the team

### DISCUSSION THREAD

**Discussion by angelomds42** [ROM-builders/temporary#22973] [General]
Hi.

My build successfully:
https://cirrus-ci.com/task/6165355651923968

Source:
https://github.com/angelomds42/android_device_motorola_devon/tree/pixelos
https://github.com/angelomds42/device_motorola_sm6225-common

Build script:
https://github.com/angelomds42/temporary/tree/devon-PixelOS-AOSP-thirteen-angelomds42

Telegram: @Knotzy06x
Github: angelomds42

**Comment by Apon77** [comment]
please follow #learn instruction from telegram group
